WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -84,26 +84,20 @@ struct ItemListView: View {
         VStack(spacing: 0) {
             headerView
 
-            if shouldShowSearchField && searchTerm.isEmpty {
-                POSRecentSearchesView(
-                    savedSearches: posModel.searchHistory(for: selectedItemListType.itemType),
-                    onSearchSelected: { search in
-                        searchTerm = search
-                    }
-                )
-                .background(Color.posSurface)
-            } else {
-                switch itemListState {
-                case .loading(let items),
-                        .loaded(let items, _),
-                        .inlineError(let items, _, _):
-                    listView(items)
-                case .error(let errorState):
-                    errorView(errorState)
-                case .empty:
-                    emptyView
-                }
-            }
+            TabView(selection: $selectedItemListType) {
+                itemListContent
+                    .tag(ItemListType.products(search: false))
```

Comment:
I didn't realize that `.tag` accepts a generic type, I though it needed to be a String/Int 💯